src/cryptography/hazmat/bindings/openssl/x509_vfy.py

```diff
@@ -29,9 +29,22 @@
 typedef ... Cryptography_STACK_OF_ASN1_OBJECT;
 
 typedef ... X509_STORE;
-typedef ... X509_STORE_CTX;
 typedef ... X509_VERIFY_PARAM;
 
+typedef struct x509_store_ctx_st {
+    X509_STORE *ctx;
+    int current_method;
+    X509 *cert;
+    Cryptography_STACK_OF_X509 *untrusted;
+    Cryptography_STACK_OF_X509_CRL *crls;
+    X509_VERIFY_PARAM *param;
+    void *other_ctx;
+    int (*verify)(struct x509_store_ctx_st *ctx);
```

Comment:
You've got an opaque typedef of X509_STORE_CTX above this, but you might get away with a forward declaration?